### PR TITLE
Fix scrolling for long threads on narrow screens

### LINKED ISSUE #7610: Message details column doesn't scroll on narrow screens
### Steps to reproduce

1. Setup up an account
2. Open a "long" message
3. Narrow the browser window

### Expected behavior

The message can be scrolled

### Actual behavior

The message content is fixed

### Mail app version

2.2

### Mailserver or service

_No response_

### Operating system

_No response_

### PHP engine version

_No response_

### Web server

_No response_

### Database

_No response_

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by GretaD**
fixes #7610

### CODE REVIEW

File: src/components/MailboxThread.vue [hunk 1]
```diff
@@ -303,10 +303,6 @@ export default {
 		background-color: var(--color-background-dark);
 	}
 }
-
-#app-content-wrapper {
-	display: flex;
```
Comment (GretaD) [inline src/components/MailboxThread.vue:308]: dead code

**Review by ChristophWurst — APPROVED** [review]
:+1: :elephant:

**Comment by ChristophWurst** [comment]
backports?

**Comment by ChristophWurst** [comment]
2.1 supersedes 2.0.

**Comment by ChristophWurst** [comment]
Does 1.14 need the fix? Otherwise 2.1 is the only backport we need

**Comment by GretaD** [comment]
> Does 1.14 need the fix? Otherwise 2.1 is the only backport we need

no, i fixed that some time ago its also released on 1.14

**Comment by ChristophWurst** [comment]
Regression https://github.com/nextcloud/mail/issues/7674 introduced by this change.

**Pull request merged**